### PR TITLE
[SofaSimulationCore] More generic DefaultVisualManagerLoop

### PR DESCRIPTION
**Pull request by damienmarchal**
Currently:
  - there must be a single VisualLoop in the scene
  - this single one must be at the root of the scene. 
  - there is no way to specify or change what is the rendered node
   
This PR allows to:
 - have multiple (possibly different) visual loop in the scene (by updating simulation::draw()). 
 - let user select which node to render (by using a SingleLink instead of a c++ attribute to store the node).
 
 This is useful as it allows to increase the separation of the different "concerns" of an object and it make explicit in the GUI the link between a node and the rendered. 

Old scenes are still possible:
 ```xml
    <DefaultVisualManagerLoop"/>
    <Node name="modelling">
         <Node name="Tripod">
         </Node>
    </Node>    
 ```

 But now we can also do:
 ```xml
    <DefaultVisualManagerLoop targetNode="@/modelling/Tripod"/>

    <Node name="modelling">
         <Node name="Tripod">
         </Node>
         <Node name="Nothing-to-Display">
         </Node>
    </Node>    
 ```
 Or even
 ```xml
    <Node name="modelling">
         <Node name="Tripod">
         </Node>
    </Node>
    
    <Node name="Simulation">
    </Node>

    <Node name="Rendering">
          <DefaultVisualManagerLoop targetNode="@/modelling/Tripod"/>
    </Node>    
 ```

Visual example:
screen

______________________________________________________

By submitting this pull request, I acknowledge that  
**I have read, understand, and agree [SOFA Developer Certificate of Origin (DCO)](https://github.com/sofa-framework/sofa/blob/master/CONTRIBUTING.md#sofa-developer-certificate-of-origin-dco)**.
______________________________________________________

**Reviewers will merge this pull-request only if**  
- it builds with SUCCESS for all platforms on the CI.
- it does not generate new warnings.
- it does not generate new unit test failures.
- it does not generate new scene test failures.
- it does not break API compatibility.
- it is more than 1 week old (or has fast-merge label).

### CODE REVIEW

**Review by alxbilger — APPROVED** [review]

File: SofaKernel/modules/SofaSimulationCore/src/sofa/simulation/DefaultVisualManagerLoop.h [hunk 1]
```diff
@@ -87,9 +87,10 @@ class SOFA_SIMULATION_CORE_API DefaultVisualManagerLoop : public sofa::core::vis
         return obj;
     }
 
+    SingleLink< DefaultVisualManagerLoop, simulation::Node, BaseLink::FLAG_STOREPATH> l_node;
```
Comment (guparan) [inline SofaKernel/modules/SofaSimulationCore/src/sofa/simulation/DefaultVisualManagerLoop.h]: Please add a Doxygen comment `//< blablabla`

File: SofaKernel/modules/SofaSimulationCore/src/sofa/simulation/DefaultVisualManagerLoop.cpp [hunk 2]
```diff
@@ -40,9 +40,9 @@ int DefaultVisualManagerLoopClass = core::RegisterObject("The simplest Visual Lo
 
 DefaultVisualManagerLoop::DefaultVisualManagerLoop(simulation::Node* _gnode)
     : Inherit()
-    , gRoot(_gnode)
+    , l_node(initLink("targetNode","Link to the topology relevant for this object"))
```
Comment (guparan) [inline SofaKernel/modules/SofaSimulationCore/src/sofa/simulation/DefaultVisualManagerLoop.cpp]: Link to the topology?

**Review by guparan — CHANGES_REQUESTED** [review]

**Review by guparan — APPROVED** [review]

**Comment by guparan** [comment]
Didn't see the Windows compilation issue yesterday. Can you look at it @damienmarchal ?

**Comment by damienmarchal** [comment]
It is an include problem that need to be fixed. I will look at it.

**Comment by damienmarchal** [comment]
[ci-build]

**Comment by damienmarchal** [comment]
[ci-build][with-scene-tests][with-regression-tests]

**Pull request merged**